wg-k8s-infra/charter.md

```diff
@@ -0,0 +1,138 @@
+# WG K8s Infra Charter
+
+This charter adheres to the conventions described in the
+[Kubernetes Charter README] and uses the Roles and Organization Management
+outlined in [sig-governance]. We are following the SIG template because we
+believe it is important this WG follow a certain level of formalism and 
+transparency that the current SIG governance template already provides.
+
+## Scope
+
+The K8s Infra Working Group is interested in the successful migration of all
+project infrastructure from Google (or elsewhere) to the CNCF, such that the
+project is able to sustainably operate itself without direct assistance from
+entities such as Google or Red Hat.
+
+It is our intent to disband once this migration is complete, with ownership
+of all code, processes and teams assigned to the appropriate SIGs. If we find
+that this is not possible, we will work with the Steering Committee and
+respective SIGs to find a more sustainable model (SIG, Committee or Team)
+
+### In scope
+
+#### Code, Binaries and Services
+
+External facing services implemented as subprojects. They often span multiple
+SIGs in terms of ownership, hence why this WG is acting as a steward of their
+migration.
+
+| Service | SIG(s) | Notes |
+| --- | --- | --- |
+| DNS | Contribex, Release | Domain Name Services for Kubernetes assets |
+| GCR | Release | Repository for Container Images |
```

Comment:
should we also mention GCB (google cloud builder), which is currently used to run releases?

Reply:
Added